File: src/test/java/ukalus/items/ScrollDescriptionResourcesTest.java
package ukalus.items;

import junit.framework.TestCase;

import java.text.MessageFormat;
import java.util.Enumeration;
import java.util.ResourceBundle;

/**
 * DOCUMENT ME!
 *
 * @author pmgremo
 */
public class ScrollDescriptionResourcesTest extends TestCase {
  public void testLoadResource() throws Exception {
    ResourceBundle bundle = ResourceBundle.getBundle(ScrollDescriptionResource.class.getName());
    Enumeration<String> keys = bundle.getKeys();

    while (keys.hasMoreElements()) {
      String key = keys.nextElement();
      System.out.printf("%s=%s%n", key, bundle.getString(key));
    }
    keys = bundle.getKeys();
    String pattern = bundle.getString(keys.nextElement());
    System.out.println(MessageFormat.format(pattern, new Object[]{0}));
    System.out.println(MessageFormat.format(pattern, new Object[]{1}));
    System.out.println(MessageFormat.format(pattern, new Object[]{30}));
  }
}

File: src/test/java/ukalus/items/ScrollDescriptionResourcesTest.kt
package ukalus.items

import junit.framework.TestCase
import java.text.MessageFormat
import java.util.*

/**
 * DOCUMENT ME!

 * @author pmgremo
 */
class ScrollDescriptionResourcesTest : TestCase() {
    fun testLoadResource() {
        val bundle = ResourceBundle.getBundle(ScrollDescriptionResource::class.java.name)
        println(bundle.inspect())

        val pattern = bundle.getString(bundle.keys.nextElement())
        println(MessageFormat.format(pattern, *arrayOf<Any>(0)))
        println(MessageFormat.format(pattern, *arrayOf<Any>(1)))
        println(MessageFormat.format(pattern, *arrayOf<Any>(30)))
    }
}